langchain/document_loaders/pdf.py

```diff
@@ -263,30 +264,10 @@ def __init__(self, file_path: str):
 
     def load(self, **kwargs: Optional[Any]) -> List[Document]:
         """Load file."""
-        import fitz
-
-        doc = fitz.open(self.file_path)  # open document
-        file_path = self.file_path if self.web_path is None else self.web_path
-
-        return [
-            Document(
-                page_content=page.get_text(**kwargs).encode("utf-8"),
-                metadata=dict(
-                    {
-                        "source": file_path,
-                        "file_path": file_path,
-                        "page_number": page.number + 1,
-                        "total_pages": len(doc),
-                    },
-                    **{
-                        k: doc.metadata[k]
-                        for k in doc.metadata
-                        if type(doc.metadata[k]) in [str, int]
-                    },
-                ),
-            )
-            for page in doc
-        ]
+
+        parser = PyMuPDFParser(text_kwargs=kwargs)
```

Comment:
this is maybe out of scope of this pr, but this taking in **kwargs is breaking the function signature right? we should probably change?

Reply:
Yeah, I'd be very happy to change it. Definitely separate PR as it's a breaking API change.